addon/services/asset-loader.js

```diff
@@ -244,6 +244,28 @@ export default Ember.Service.extend({
         link.href = uri;
 
         document.head.appendChild(link);
+
+        // In case the browser doesn't fire onload/onerror events, we poll the
+        // the list of stylesheets to see when it loads
+        function checkSheetLoad() {
+          const stylesheets = document.styleSheets;
```

Comment:
Is this needed for every browser? Does 'load' work correctly in some browsers? Im wondering if this should be a fallback instead of always polling. 

Does this fix the render blocking? Do you think we may need the media="x" trick until it is loaded?